src/NServiceBus.Testing/Saga.cs

```diff
@@ -345,7 +345,7 @@ public Saga<T> AssertSagaCompletionIs(bool complete)
         /// </summary>
         public Saga<T> ExpectTimeoutToBeSetIn<TMessage>(Func<TMessage, TimeSpan, bool> check = null)
         {
-            testableMessageHandlerContext.ExpectedInvocations.Add(new ExpectDelayDeliveryWith<TMessage>(check));
+            testContext.ExpectedInvocations.Add(new ExpectDelayDeliveryWith<TMessage>(check));
```

Comment:
would work the same. Since it's internal now, I don't really see this making a huge difference.